Assets/_BForBoss/_Character/Scripts/FirstPersonPlayer.InputSettings.cs

```diff
@@ -15,7 +15,7 @@ public interface IInputSettings
 
     public partial class FirstPersonPlayer : IInputSettings
     {
-        private struct PlayerPrefKey
+        private partial struct PlayerPrefKey
```

Comment:
This as well, I'm wondering if we might want a cleaner way of handling player prefs

Reply:
https://perigongames.atlassian.net/browse/BFB-75

Yup clean up here